docs/src/devdocs/dev_install.md

```diff
@@ -1,74 +1,201 @@
+
+NOTE: (I am on OSX, date is 4 Nov 2016, use common sense to update version numbers etc.)
+
+* If you want to make sure of fresh start, you might want to start by removing any existing Julia or Atom (Note: you shouldn't need to do this, you can have multiple Julia versions side-by-side):
 
 ```shell
-apm install https://github.com/JunoLab/atom-ink
-apm install https://github.com/JunoLab/atom-julia-client
+rm -rf ~/.julia
+rm -rf /Applications/Julia-0.5.app
+
+# https://discuss.atom.io/t/how-to-completely-uninstall-atom-for-mac/9084/34
+rm     /usr/local/bin/atom 
+rm     /usr/local/bin/apm
+rm -rf /Applications/Atom.app
+rm     ~/Library/Preferences/com.github.atom.plist
+rm -rf ~/Library/Application\ Support/com.github.atom.ShipIt
+rm -rf ~/Library/Application\ Support/Atom
+rm -rf ~/Library/Saved\ Application\ State/com.github.atom.savedState
+rm -rf ~/Library/Caches/com.github.atom
+rm -rf ~/Library/Caches/Atom
 ```
 
-It's a good idea to keep these up to date by running `Pkg.update()` in Julia and
-syncing the package repos every now and then, which will be in `~/.atom/packages/julia-client`
-and `~/.atom/packages/ink`. Note that this installation method for the Atom packages
-will allow them to be update each commit via the Atom package manager.
+* Install the current ***stable*** Julia (command line version) [**NOT bleeding-edge**] from http://julialang.org/downloads/  
+  Drag .app into Applications folder  
+  Run
 
-### Bleeding Edge Installs for the Julia Packages
+* Install Atom from https://atom.io/  
+  Drag .app into Applications folder  
+  Run
 
-The Julia packages can be installed by either cloning the repository or though
-the standard installation (`Pkg.add(PkgName)`) followed by checking out the master branch
-(`Pkg.checkout("PkgName")`. The following code will check out the relevant packages:
+Ok, you should have Atom and a `Julia>` Julia-prompt side-by-side.
 
-```julia
-Pkg.checkout("Atom")
-Pkg.checkout("Blink")
-Pkg.checkout("CodeTools")
-Pkg.checkout("DevTools")
-Pkg.checkout("Hiccup")
-Pkg.checkout("Juno")
-Pkg.checkout("LNR")
-Pkg.checkout("Media")
+`Julia> Pkg.status()` from Julia prompt should say no packages installed!  `Packages` menu drop down in Atom shouldn't have "Julia" yet.
+
+* Use Atom's `apm` package manager to install `atom-julia-client` (Julia IDE for Atom) which makes use of `ink` (toolkit for building IDEs in Atom):
+
+```shell
+apm install https://github.com/JunoLab/atom-ink           # ~/.atom/packages/ink
+apm install https://github.com/JunoLab/atom-julia-client  # ~/.atom/packages/julia-client
 ```
 
-### Bleeding Edge Features (Plotting, Debugging, etc.)
+You can observe this creates a local mirror of each GitHub repo.  
+(Note: You can sync all local repos with `Pkg.update()`
+
+If you now relaunch Atom you will see the `Packages` menu drop down now has a "Julia" entry.
+  `Packages -> Julia -> Open Console`
```

Comment:
I would get rid of this: if they want to do the dev install, they should know how to open the console. Otherwise I think you'd be leading someone where they shouldn't be going.